NitroxPatcher/Patches/DayNightCycle_OnConsoleCommand_day_Patch.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using Harmony;
+using System.Reflection;
+
+namespace NitroxPatcher.Patches
+{
+    class DayNightCycle_OnConsoleCommand_day_Patch : NitroxPatch
+    {
+        public static readonly Type TARGET_CLASS = typeof(DayNightCycle);
+        public static readonly MethodInfo TARGET_METHOD = TARGET_CLASS.GetMethod("OnConsoleCommand_day", BindingFlags.Instance | BindingFlags.NonPublic);
+
+        public static bool Prefix()
+        {
+            return false;
```

Comment:
We can block these but should we send the corresponding packets?  (if the users doesn't have permission the server will just let them know.